README.md

```diff
@@ -32,11 +32,15 @@ The repository is organized as follows:
 
     * `plot_acceleration_from_sequential_learning.py`: Script to aggregate results from the `sequential_learning.py` script, calculate and plot statistics related to acceleration from SL over a baseline.
 
-        If run as-is, the script reproduces the 3-paneled FIG.~5 in the main text.
+        If run as-is, the script reproduces the 3-paneled FIG. 5 in the main text.
 
     * `plot_acceleration_from_sequential_learning__ALL_ACQ.py`: Similar to the previous script; plots and compares statistics from all acquisition functions considered (MLI, MU, random, space-filling) for all SL tasks.
 
-        If run as-is, the script reproduces the 3-paneled FIG.~S1 in the Supplementary Information.
+        If run as-is, the script reproduces the 3-paneled FIG. S1 in the Supplementary Information.
+
+    * `plot_levels_of_automation.py`: Script for plotting the cumulative time  of executing the DFT pipeline at varying levels of automation.
```

Comment:
Not for this PR, but please add the above line to the `supplementary.tex` file in the paper as well.

Reply:
thanks -- good catch. done